picard/plugin3/api.py

```diff
@@ -634,7 +633,7 @@ def register_cover_art_processor(self, processor_class: Type[ImageProcessor]) ->
 
     # File formats
     def register_format(self, format: Type[File]) -> None:
-        return register_format(format)
+        return self.tagger.format_registry.register(format)
```

Comment:
`tagger` is the exposed property, but here you can use the private attribute  `self._tagger`

Reply:
@zas I fixed this in the initial commit and rebased